app/controllers/api/v0/admin/auctions_controller.rb

```diff
@@ -0,0 +1,12 @@
+class Api::V0::Admin::AuctionsController < ApiController
+  before_filter :require_admin
```

Comment:
I hear you, but we have some API controllers that aren't admin and some admin controllers that aren't API